apps/web/playwright/booking-pages.e2e.ts

```
@@ -655,9 +655,12 @@ test.describe("Event type with disabled cancellation and rescheduling", () => {
   });
 
   test("Should prevent cancellation and show an error message", async ({ page }) => {
+    const reponse = await page.request.get("/api/csrf");
+    const { csrfToken } = await response.json();
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)